spec/support/fixtures/vcr_cassettes/audit_trail/get_events.yml

```diff
@@ -0,0 +1,61 @@
+---
+http_interactions:
+  - request:
+      method: get
+      uri: https://api.workos.com/events
+      body:
+        encoding: US-ASCII
+        string: ""
+      headers:
+        Content-Type:
+          - application/json
+        Accept-Encoding:
+          - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+        Accept:
+          - "*/*"
+        User-Agent:
+          - WorkOS; ruby/2.7.1; x86_64-darwin19; v0.3.3
```

Comment:
You'll need to bump the version here as well—this will fail somewhere down the line

Reply:
Wild! Curious to hear which cases they would fail I thought the user agent in vcr was largely incidental. I've gone ahead and done so. I notice all of the other cassettes have older versions too do we need to update them as well?

Reply:
Oh maybe it doesn't 🤔 

I recall versioning causing failures but I'm not sure where